db/fixtures/notification_types.yml

```diff
@@ -174,3 +174,83 @@
   :expires_in: 24.hours
   :level: :error
   :audience: global
+- :name: vm_destroy_success
+  :message: 'Destroying Instance %{subject} completed successfully.'
+  :expires_in: 24.hours
+  :level: :success
+  :audience: global
+- :name: vm_destroy_error
+  :message: 'Destroying Instance %{subject} failed: %{error_message}'
+  :expires_in: 24.hours
+  :level: :error
+  :audience: global
+- :name: cloud_volume_attach_success
+  :message: 'Attaching Volume %{subject} to Instance %{instance_name} completed successfully.'
+  :expires_in: 24.hours
+  :level: :success
+  :audience: global
+- :name: cloud_volume_attach_error
+  :message: 'Attaching Volume %{subject} to Instance %{instance_name} failed: %{error_message}'
+  :expires_in: 24.hours
+  :level: :error
+  :audience: global
+- :name: cloud_volume_detach_success
+  :message: 'Detaching Volume %{subject} from Instance %{instance_name} completed successfully.'
+  :expires_in: 24.hours
+  :level: :success
+  :audience: global
+- :name: cloud_volume_detach_error
+  :message: 'Detaching Volume %{subject} from Instance %{instance_name} failed: %{error_message}'
+  :expires_in: 24.hours
+  :level: :error
+  :audience: global
+- :name: cloud_volume_create_success
+  :message: 'Creating Volume %{volume_name} completed successfully.'
```

Comment:
why no `subject` here?

Reply:
b/c this called when creating that resource - and we do not have it so we cannot pass it into notification.

I can use `subject` name as it would just have `volume_name` value in it for the sake of more consistent code for calling.